analyzer/analyzer_test.go

```diff
@@ -32,3 +32,25 @@ func TestAnalyzer(t *testing.T) {
 		})
 	}
 }
+
+func TestAnalyzer_Rules_RemoteConfig(t *testing.T) {
+	test := "rules"
+	url := "https://raw.githubusercontent.com/quasilyte/go-ruleguard/master/rules.go"
```

Comment:
better to run server locally, so test will be isolated

Reply:
Great idea! I didn't know about httptest package before. Beautiful thing. Fixed :+1: